pkg/image/prune/prune.go

```diff
@@ -55,6 +55,9 @@ const (
 	// ReferencedImageLayerEdgeKind defines an edge from an ImageStreamNode or an
 	// ImageNode to an ImageComponentNode.
 	ReferencedImageLayerEdgeKind = "ReferencedImageLayer"
+
+	// TODO choose final name and promote to API
+	protectionAnnotation = "image.openshift.io/prune-immunity"
```

Comment:
@openshift/api-review ptal (yes David I know it's you, but also others 😉)

Reply:
> @openshift/api-review ptal (yes David I know it's you, but also others )

Fair. :)